code/__DEFINES/cult_defines.dm

```diff
@@ -10,7 +10,7 @@
 #define RUNE_COLOR_EMP "#4D94FF"
 #define RUNE_COLOR_SUMMON "#00FF00"
 
-#define IS_SACRIFICE_TARGET(A) SSticker?.mode?.cult_team?.is_sac_target(A)
+#define is_sacrifice_target(A) SSticker.mode?.cult_objs.is_sac_target(A)
```

Comment:
You seem to have an accidental revert of #24379 mixed in here, please fix.

Reply:
This has been fixed.